docs/prebuilt.md

```diff
@@ -48,8 +49,10 @@ The prebuilt binaries in `rv32emu-prebuilt` are built from the following reposit
 - `pi` : See [tests/pi.c](/tests/pi.c)
 - `puzzle` : See [tests/puzzle.c](/tests/puzzle.c)
 - `qrcode` : See [tests/qrcode.c](/tests/qrcode.c)
+- `quake` : See [sysprog21/quake-embedded](https://github.com/sysprog21/quake-embedded)
```

Comment:
Mention RV32F explicitly.

Reply:
The current prebuilt binaries are built with `-march=rv32im` option. I'd like to keep `quake` been built without F extension for the consistency.

Reply:
SciMark2 should be built in RV32F since it reports a composite score in approximate Mflops (Millions of floating point operations per second).

Reply:
> SciMark2 should be built in RV32F since it reports a composite score in approximate Mflops (Millions of floating point operations per second).

Got it. Should we also build `quake` with `rv32f`? I feel the execution is smooth when building with `rv32im`.

Reply:
> Got it. Should we also build `quake` with `rv32f`? I feel the execution is smooth when building with `rv32im`.

Quake needs to be compiled with RV32IMF. Otherwise, it is unlikely to function properly.

Reply:
I found `quake` is already built with RV32IMF within its Makefile. I'll update the document later.